tests/ttnn/unit_tests/operations/test_clone.py

```diff
@@ -10,11 +10,80 @@
 from loguru import logger
 
 from tests.ttnn.unit_tests.operations.test_utils import (
-    to_cpu,
-    to_npu,
+    get_compute_kernel_options,
+    compute_kernel_options,
+    compute_kernel_ids,
 )
 
 
+def to_ttnn(
+    torch_tensor,
+    device,
+    *,
+    ttnn_layout=ttnn.TILE_LAYOUT,
+):
+    """
+    Converts a PyTorch tensor to a TTNN tensor with the specified device and layout.
+
+    This function handles tensors with 0 or 1 dimensions by reshaping them to have at least 2 dimensions:
+    - If the input tensor is a scalar (0-dimensional), it is reshaped to shape `[1, 1]`.
+    - If the input tensor is 1-dimensional, it is reshaped to shape `[1, length]`.
+
+    Parameters:
+    torch_tensor: torch.Tensor or None
+        The PyTorch tensor to convert. If None, the function returns None.
+    device: ttnn.device
+        The device to which the TTNN tensor will be moved.
+    ttnn_layout: ttnn.TILE_LAYOUT or ttnn.ROW_MAJOR_LAYOUT, optional
+        The desired layout for the TTNN tensor. Defaults to `ttnn.TILE_LAYOUT`.
+
+    Returns:
+    ttnn.Tensor or None
+        The converted TTNN tensor, with the specified layout and moved to the specified device.
+    """
+    if torch_tensor is None:
+        return None
+    if len(torch_tensor.shape) == 0:
+        torch_tensor = torch.reshape(torch_tensor, shape=[1, 1])
+    if len(torch_tensor.shape) == 1:
+        torch_tensor = torch.reshape(torch_tensor, shape=[1, len(torch_tensor)])
```

Comment:
Can this be a single call from_touch with device and layout params? I expect they to work

Reply:
Same for the below to_torch

Reply:
My answer is: Yes. I noticed this while resolving the following issue:
https://github.com/tenstorrent/tt-metal/issues/13804
I will recheck all test files for consistency in the PR related to that issue.

Reply:
Updated to use `to_torch`, `from_torch` directly.